cart.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MetaKart</title>
+    <link rel="stylesheet" href="https://hardcore-aryabhata-201d65.netlify.app/Css/main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="top-navbar">
+        <div class="top-navbar-icon">
+            <img src="https://bit.ly/3v2WZoY"/>
+            <div class="title">MetaKart</div>
+        </div>
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
```

Comment:
you can give this label a class and then apply style to it (avoid inline styling)

Reply:
Yeah....